about.html

```diff
@@ -0,0 +1,153 @@
+          and outdoor activities.
+          Amateur soccer supporters, Sports lovers, their families and friends, etc, are all invited to
+          the 5-day event. Various people active in the field of soccer will gather in one place to
+          share richer and more dynamical experiencies. We hope that it will be an opportunity
+          to share our thoughts on creating an ecosystem.</p>
+        <p class="about2 onlydesktop">In particular, this year's FICA Tournament is <br>waiting for the
+        participation of various teams who want to spread the value of sharing, free culture,
+        and outdoor activities.<br>
+        Amateur soccer supporters, Sports lovers, their families and friends, etc, are all invited to
+        the 5-day event. <br>Various people active in the field of soccer will gather in one place to
+        share richer and more dynamical experiencies.<br> We hope that it will be an opportunity
+        to share our thoughts on creating an ecosystem.</p>
+          <div class="centerimg"><img src="img/about_icon.png" width=39 height=10 alt=""></div>
+        <p class="about3 tocenter2">September 14 - 18, 2022 (Wednesday - Sunday)
+          <br>See you at the FICA Soccer Tournament!</p>
+      </div>
+      <p class="about4 tocenter2">Do you have any other questions about FICA Soccer Tournament 2022?
+        <br class="onlymobile">&nbsp;&nbsp;&nbsp;
+        <a href="mailto:acolombo1@yahoo.com" class="asnormal">acolombo1@yahoo.com</a></p>
+      <div class="special onlymobile"><hr class="hr4"></div>
+      <div class="mini-spacer onlydesktop">&nbsp;</div>
+    </div>
+  </section>
+
+  <section class="about-section2-wrapper">
+    <div class="about-section2">
+      <h2 class="not-appear">FICA Global 2022 Logo Contest</h2>
+      <p class="about5 tocenter2">FICA Global 2022 Logo Contest</p>
+      <hr class="hr2">
+      <p class="about2 tocenter2">The winner of the logo contest, which was held from June 8th to July 7th,
+        was decided through online voting and evaluation by the judges.<br class="onlydesktop">
+        After consultation with the winning designer, the logo of FICA Tournament 2022, which was optimized
+        for use on the website and promotional materials, is as follows.</p>
+      <p class="tocenter2 margintopbottom"><img id="logo1" src="img/logo_big.png" width="260" height="102" Alt="Logo"></p>
+      <p class="about2 tocenter2">Aldo Colombo of Argentina, who designed the winning piece, explains the
+         meaning of the logo as follows:</p>
+      <p class="about2 tocenter2">"</p>
+      <p class="about2 tocenter2">The tri-colored ball symbolizes the integration of different
+        cultures and ethnicities, united in a sigle world of football. The
+        mimalist design symbolizes purity, harmony, wholeness and infinity.</p> 
+      <p class="about2 tocenter2">"</p>
+    </div>
+  </section>
+
+  <hr class="hr5">
+
+  <section class="about-section3-wrapper">
```

Comment:
- [ ] Your bottom section is almost there! However, we still need to add text over the two images like the template:

Template:
![Captura de pantalla 2022-04-14 a la(s) 10 23 42 p  m](https://user-images.githubusercontent.com/83717285/163513562-82434979-fe1b-477c-8182-d6fdc244fa66.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 10 24 48 p  m](https://user-images.githubusercontent.com/83717285/163513585-329a0192-a08e-4848-a847-8a8c0b6a6877.png)